client/plots/sampleScatter.js

```diff
@@ -314,7 +328,7 @@ export async function getPlotConfig(opts, app) {
 		}
 		// may apply term-specific changes to the default object
 		const result = copyMerge(config, opts)
-		return result
+		if (result.term0?.q?.mode == 'continuous' && !app.hasWebGL()) return result
```

Comment:
@airenzp it's returning undefined for premade plots

Reply:
@airenzp this has to throw here, won't be able to recover a save session with 3d plot when the browser does not support WebGL

Reply:
Yes, added the throw now, thanks for the heads up.